packages/shared/src/lib/flow-run/execution/flow-execution.ts

```diff
@@ -62,6 +62,8 @@ const BaseExecutionResponse = {
     tags: Type.Optional(Type.Array(Type.String())),
     error: Type.Optional(FlowError),
     stopResponse: Type.Optional(StopResponse),
+    triggerPayload: Type.Optional(Type.String()),
```

Comment:
We already do have triggerPayload in the steps, I don't think we need it for here.

For `resumePayload` it make sense, but did we try to test If the flow is paused and failed (due to quota exceeded) can we retry it?